src/backend/people/api_urls.py

```diff
@@ -21,6 +21,11 @@
     basename="team_accesses",
 )
 
```

Comment:
could we get inspiration from [here](https://github.com/openfun/joanie/blob/538d737f2f4090a7aac36df694534178b2cd3181/src/backend/joanie/client_urls.py#L57C1-L57C22).
to avoid duplicating the `re_path` and have a unique and dedicated router for team related routes.

Reply:
I didn't think that through, it's a good idea !